src/diffcalc/hkl/calc.py

```diff
@@ -486,13 +573,7 @@ def __tidy_degenerate_solutions(
                 )
                 print("            original:", pos)
             newpos = Position(
-                pos.mu,
-                pos.delta,
-                pos.nu,
-                desired_eta,
-                pos.chi,
-                pos.phi - eta_diff,
-                False,
+                pos.mu, pos.delta, pos.nu, desired_eta, pos.chi, pos.phi - eta_diff
```

Comment:
Did black do this? It might be worth making a separate PR to:

- Run black on the whole codebase
- Add a black CI job